lib/msal-common/test/response/ResponseHandler.spec.ts

```diff
@@ -110,6 +110,7 @@ const testServerTokenResponse = {
         scope: TEST_CONFIG.DEFAULT_SCOPES.join(" "),
         expires_in: TEST_TOKEN_LIFETIMES.DEFAULT_EXPIRES_IN,
         ext_expires_in: TEST_TOKEN_LIFETIMES.DEFAULT_EXPIRES_IN,
+        refresh_in: TEST_TOKEN_LIFETIMES.DEFAULT_REFESH_IN,
```

Comment:
I think it's better to have another response, because you want tests that cover both cases: 

- refresh_in is present in the response
- refresh_in is not present